packages/react-composites/tests/browser/chat/app/FakeAdapterApp.tsx

```diff
@@ -28,24 +36,61 @@ const params = Object.fromEntries(urlSearchParams.entries());
  */
 export const FakeAdapterApp = (): JSX.Element => {
   // Required params
-  const fakeChatAdapterModel = JSON.parse(
-    verifyParamExists(params.fakeChatAdapterModel, 'fakeChatAdapterModel')
-  ) as ChatAdapterModel;
+  const fakeChatAdapterArgs = JSON.parse(
+    verifyParamExists(params.fakeChatAdapterArgs, 'fakeChatAdapterArgs')
+  ) as FakeChatAdapterArgs;
 
   // Optional params
   const useFrLocale = Boolean(params.useFrLocale);
+  const fileSharingEnabled = Boolean(params.fileSharingEnabled);
```

Comment:
Instead of adding query args for all these cases, can the test create the appropriate `Model` in some of these cases and pass the serialized `Model` as a single query argument?

The fewer the layers between the test and the eventual model / adapter, the easier it is to understand and add tests.

I know we had some trouble earlier importing the types in the test code. Can we give that another look. We seem to be bending over backwards here to pass all this test data, and it makes working with the e2e tests much harder.